services/userService.js

```diff
@@ -0,0 +1,74 @@
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+
+const { userDao } = require("../models");
+
+const hashPassword = async (plaintextPassword) => {
+  const saltRounds = 10;
+
+  return await bcrypt.hash(plaintextPassword, saltRounds);
```

Comment:
```suggestion
  return bcrypt.hash(plaintextPassword, saltRounds);
```

promise를 바로 return하는 경우에는 await를 작성하지 않아도 동작이 동일할 것 같습니다! 한번 테스트해주세요~